packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx

```diff
@@ -244,16 +244,19 @@ export const CreateCluster = () => {
             </StyledDocsLinkContainer>
           </StyledRegionSelectStack>
           <Divider sx={{ marginTop: 4 }} />
-          <Select
-            onChange={(selected: Item<string>) => {
+          <Autocomplete
+            isOptionEqualToValue={(option, value) =>
+              option.value === value.value
+            }
+            onChange={(_, selected: Item<string>) => {
               setVersion(selected);
             }}
+            disableClearable={!!version}
             errorText={errorMap.k8s_version}
-            isClearable={false}
             label="Kubernetes Version"
             options={versions}
             placeholder={' '}
-            value={version || null}
+            value={version ?? null}
```

Comment:
We should drive `value` from our array of `versions`

```suggestion
            value={versions.find((v) => v.value === version) ?? null}
```

By doing this we can remove `isOptionEqualToValue`.